### PR TITLE
Added support for Edge Stable channel

### PR DESCRIPTION
**Pull request by mustjab**
Now that Edge Stable build has shipped, we'd like to enable runs on it on a regular basis ( #21216 ). I set it to run daily, but let me if you prefer to run this less frequently.

### CODE REVIEW

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -239,6 +240,42 @@ jobs:
...
+  - template: tools/ci/azure/system_info.yml
+  - template: tools/ci/azure/checkout.yml
+  - template: tools/ci/azure/install_python.yml
+  - template: tools/ci/azure/pip_install.yml
+    parameters:
+      packages: virtualenv
+  - template: tools/ci/azure/install_certs.yml
+  - template: tools/ci/azure/install_edge.yml
+    parameters:
+      channel: dev
```
Comment (foolip) [inline .azure-pipelines.yml]: This should say stable

File: tools/ci/azure/install_edge.yml [hunk 2]
```diff
@@ -4,6 +4,26 @@ parameters:
 # Should match https://web-platform-tests.org/running-tests/chrome.html
 # Just replace chrome with edgechromium
 steps:
+- ${{ if eq(parameters.channel, 'stable') }}:
+  - powershell: |
+      $edgeInstallerName = 'MicrosoftEdgeSetup.exe'
+      # Link to Canary channel installer
```
Comment (foolip) [inline tools/ci/azure/install_edge.yml]: Stable?

File: .azure-pipelines.yml [hunk 3]
```diff
@@ -239,6 +240,42 @@ jobs:
...
+  - template: tools/ci/azure/update_hosts.yml
+  - template: tools/ci/azure/update_manifest.yml
+  - script: python ./wpt run --yes --no-manifest-update --no-restart-on-unexpected --no-fail-on-unexpected --install-fonts --this-chunk $(System.JobPositionInPhase) --total-chunks $(System.TotalJobsInPhase) --chunk-type hash --log-wptreport $(Build.ArtifactStagingDirectory)/wpt_report_$(System.JobPositionInPhase).json --log-wptscreenshot $(Build.ArtifactStagingDirectory)/wpt_screenshot_$(System.JobPositionInPhase).txt --log-mach - --log-mach-level info --channel stable edgechromium
+    displayName: 'Run tests (Edge Stable)'
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish results'
+    inputs:
+      artifactName: 'edge-stable-results'
+  - template: tools/ci/azure/cleanup_win10.yml
```
Comment (foolip) [inline .azure-pipelines.yml]: Is any cleanup change needed?

File: tools/ci/azure/install_edge.yml [hunk 4]
```diff
@@ -4,6 +4,26 @@ parameters:
...
+      Start-BitsTransfer -Source 'https://go.microsoft.com/fwlink/?linkid=2108834&Channel=Stable&language=en' -Destination MicrosoftEdgeSetup.exe
+      if (-not (Test-Path $edgeInstallerName)) {
+        Throw "Failed to download Edge installer to $edgeInstallerName."
+      }
+      cmd /c START /WAIT $edgeInstallerName /silent /install
+      $edgePath = "$env:systemdrive\Program Files (x86)\Microsoft\Edge\Application"
+      if (Test-Path $edgePath) {
+        Write-Host "##vso[task.prependpath]$edgePath"
+        Write-Host "Edge Canary installed at $edgePath."
```
Comment (foolip) [inline tools/ci/azure/install_edge.yml]: Stable

**Review by foolip — COMMENTED** [review]
Can you trigger a full run and see that the results end up in wpt.fyi as expected?

File: .azure-pipelines.yml (hunk 3 above)
Comment (mustjab) [reply .azure-pipelines.yml]: No, cleanup code already handles stable channel removal.

File: .azure-pipelines.yml (hunk 1 above)
Comment (mustjab) [reply .azure-pipelines.yml]: Fixed

File: tools/ci/azure/install_edge.yml (hunk 2 above)
Comment (mustjab) [reply tools/ci/azure/install_edge.yml]: Scheduled run passed: https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=40402&view=results

**Comment by foolip** [comment]
The results of https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=40402&view=results showed up at https://wpt.fyi/results/?run_id=422580001 with the right labels, so everything seems to be working.

Ignoring the "Azure Pipelines (infrastructure/ tests: macOS)" failure.

**Pull request merged**